core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -3847,6 +3847,15 @@ object Stream extends StreamLowPriority {
       else go(Nil, n, self)
     }
 
+    /** Like [[uncons]] but skips over the empty chunk. */
```

Comment:
```suggestion
    /** Like [[uncons]] but skips over empty chunks, pulling until it can emit the first non-empty chunk. */
```

What do you think about this alternate wording? I think it's helpful to be very explicit in the method docs since fs2 has so many operations